src/client/Microsoft.Identity.Client/Platforms/Android/Broker/AndroidBroker.cs

```diff
@@ -69,10 +98,6 @@ public bool IsBrokerInstalledAndInvokable()
             try
             {
                 await _brokerHelper.InitiateBrokerHandshakeAsync(_parentActivity).ConfigureAwait(false);
-
-                // todo: needed? 
-                // brokerPayload[BrokerParameter.BrokerAccountName] = AndroidBrokerHelper.GetValueFromBrokerPayload(brokerPayload, BrokerParameter.Username);              
```

Comment:
would have been great with a comment on why this was removed. apparently not needed, just not sure why